frontend/src/helpers/index.ts

```diff
@@ -352,6 +352,14 @@ function loadTableSettings(tableId: string): { id: string; show: boolean }[] {
   return settings;
 }
 
+/**
+ * @returns true if the websocket multiplexer is enabled.
+ * defaults to false
+ */
+export function getWebsocketMultiplexerStatus(): boolean {
```

Comment:
`getWebsocketMultiplexerEnabled` imho would be a better name. comment should mention that this is a temporary feature toggle